docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -406,29 +406,18 @@ this person as a dependent.
 
 **For simulant under 18:**
```

Comment:
if this edit would be accurate, would you mind updating this to "simulant under 18 living in non-GQ" (or something similar?)

Reply:
Good callout!